HealthBarForSavesScreen.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.IO;
+using CodeMonkey;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class HealthBarForSavesScreen : MonoBehaviour
+{
+    private const float DAMAGED_HEALTH_SHRINK_TIMER_MAX = 25f;
+    
+    public Slider fillSlider;
+    private Slider shieldSlider;
+    private float damagedHealthShrinkTimer;
+    private SaveManager _saveManager;
+
+    private int healthBarNumber;
+    
+    private bool _saveDoesNotExist;
+    private string folderName;
+
+    public int currentSave;
+
+    public PlayerStats stats;
+
+    /*public Color darkRed = new Color(172, 0, 14, 255);
+    public Color brightRed = new Color(255, 0, 14, 255);*/
+    private Image _fillImage;
```

Comment:
```suggestion
    public Slider fillSlider;
    public int currentSave;
    public PlayerStats stats;

    private Slider shieldSlider;
    private float damagedHealthShrinkTimer;
    private SaveManager _saveManager;
    private int healthBarNumber;
    private bool _saveDoesNotExist;
    private string folderName;
    private Image _fillImage;
```

Here, notice that I have group by access modifier, and if possible by type, this is code clean practice.
I have read in C# code that private fields or variables, they prefix that with '_', in my opinion, this does not make sense.

Reply:
So I remove "_"?, but Rider says it has to be with a "_" prefix.